.tekton/testing-farm.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: testing-farm
+spec:
+  description: Initiate testing-farm test given a list of container images
+  params:
+    - name: SNAPSHOT
+      description: A list of container images that should undergo testing
+    - name: GIT_URL
+      description: URL of the GIT repository that contains the tests.
+    - name: GIT_REF
+      default: "main"
+      description: Branch of the git repository used containing the tests
+    - name: COMPOSE
+      default: "Fedora-Rawhide"
+      description: Compose to use for the system-under-test.
```

Comment:
This is a virtual machine under which we run podman, it's not really system-under-test no?  (Other than I guess it contains the podman version)